### PR TITLE
Document zwave_js Multilevel Switch

### PR DESCRIPTION
**Pull request by reubenbijl**
Added feature in ZwaveJS 9
https://github.com/home-assistant/core/pull/68830#pullrequestreview-950921456

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/70470
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -341,6 +341,37 @@ These are notification events fired by devices using the Notification command cl
 }
 ```
 
+#### MultilevelSwitch Notification Command Class
```
Comment (raman325) [inline source/_integrations/zwave_js.markdown]:
```suggestion
#### Multilevel Switch Command Class
```

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -341,6 +341,37 @@ These are notification events fired by devices using the Notification command cl
...
+    "node_id": 1,
+    "home_id": 974823419,
+    "device_id": "2f44f0d4152be3123f7ad40cf3abd095",
+    "command_class": 38,
+    "command_class_name": "Multilevel Switch",
+    "type": 4,
+    "parameters": "up"
+},
+```
+##### Stop Level Change
```
Comment (raman325) [inline source/_integrations/zwave_js.markdown]:
```suggestion
\```

##### Stop Level Change
```

Comment (raman325) [reply source/_integrations/zwave_js.markdown]: Remove the slash, it was the only way to make this a commitable suggestion

File: source/_integrations/zwave_js.markdown [hunk 3]
```diff
@@ -341,6 +341,38 @@ These are notification events fired by devices using the Notification command cl
...
+    "domain": "zwave_js",
+    "node_id": 1,
+    "home_id": 974823419,
+    "device_id": "2f44f0d4152be3123f7ad40cf3abd095",
+    "command_class": 38,
+    "command_class_name": "Multilevel Switch",
+    "type": 4,
+    "parameters": "up"
+},
+\```
```
Comment (raman325) [inline source/_integrations/zwave_js.markdown:360]:
```suggestion
```
```

**Review by raman325 — CHANGES_REQUESTED** [review]
please remove the backslash I added and then this should be good to go

**Review by raman325 — DISMISSED** [review]
Fixed and approved, thanks!

**Review by reubenbijl — COMMENTED** [review]
Thanks, sorry, I read it correctly, and then accidentally accepted the whole commit.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**